crates/cli/tests/benchmarks/platform/host.zig

```diff
@@ -123,26 +121,15 @@ pub fn main() !u8 {
         roc_dealloc(raw_output, @alignOf(u64));
     }
 
-    var timer = std.time.Timer.start() catch unreachable;
-
     roc__mainForHost_1_exposed_generic(output);
 
     const closure_data_pointer = @as([*]u8, @ptrCast(output));
 
     call_the_closure(closure_data_pointer);
 
-    const nanos = timer.read();
-    const seconds = (@as(f64, @floatFromInt(nanos)) / 1_000_000_000.0);
-
-    stderr.print("runtime: {d:.3}ms\n", .{seconds * 1000}) catch unreachable;
-
```

Comment:
We now compare the expected output from both stdout and stderr, instead of just stdout as previously, to verify it behaves as expected. Removing the timing information from zig test platforms simplifies the comparison -- and this wasn't being used for anything. There are now other more mature examples of platform implementation for people interested to use as examples.